### ISSUE TITLE
Code completion for CUDA files

### ISSUE THREAD

**Issue by rusty1s** [ncm2/ncm2-pyclang#20]
Hi,
I would like to use this plugin in CUDA files (`.cuh` and `.cu`), but it seems that filetypes are kinda hardwired into the (source code)[https://github.com/ncm2/ncm2-pyclang/blob/master/autoload/ncm2_pyclang.vim#L55]. Do you have any hint on how to use this plugin with cuda files?

**Comment by banminkyoz** [comment]
Agree. There should be configuration to use this plugin with other files type. I also do want to use this with .ino files. Is there any workaround for this @roxma sama :smile:

**Comment by roxma** [comment]
Hi, I'm too busy to add & test this feature. I'm not familiar with CUDA either.

PR is welcome.
